custom-routes-vis/src/main/webapp/script.js

```diff
@@ -143,9 +156,140 @@ function hideMarker(markerName) {
   document.getElementById('generate-routes').style.display = 'none';
 }
 
+/** Removes all routes from the DOM. */
+function clearRoutes() {
+  for (let i = 0; i < displayedRoutes.length; i++) {
+    displayedRoutes[i].setMap(null);
```

Comment:
Could you use a forEach loop here?

Reply:
Yeah I think I can but what is the advantage of a forEach loop?

Reply:
It's cleaner I think.